tests/example.at

```diff
@@ -25,11 +25,13 @@ POSITIVE_TEST([cliquer],[example])
 POSITIVE_TEST([comp-parametrized-inherit],[example])
 POSITIVE_TEST([comp-parametrized-multilvl],[example])
 POSITIVE_TEST([comp-parametrized],[example])
+POSITIVE_TEST([counter],[example])
 POSITIVE_TEST([degree],[example])
 POSITIVE_TEST([dfa],[example])
 POSITIVE_TEST([dfa_live_vars],[example])
 POSITIVE_TEST([dfa_min],[example])
 POSITIVE_TEST([dfa_summary_function],[example])
+POSITIVE_TEST([dfa_parse],[example])
```

Comment:
Not quite alphabetical order

Reply:
Fixed in feb692275c6358a67396810e843d583af4e2f7ec